cases/aeria/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import type { Collection } from '@aeriajs/types';
+
+declare global {
+  type Collections = Record<string, Collection>;
+}
```

Comment:
Could you please add a comment about why this is needed for future selves :)

Reply:
Actually, it seems that those types are not necessary at all, good catch thank you :)